akka-actor/src/main/scala/akka/io/Dns.scala

```diff
@@ -66,12 +92,21 @@ object Dns extends ExtensionId[DnsExt] with ExtensionIdProvider {
         addresses.iterator.collect { case a: Inet6Address => a }.to(immutable.IndexedSeq)
       Resolved(name, ipv4, ipv6)
     }
+
+    @deprecated("cached(DnsProtocol.Resolve)", "2.6.0")
+    def apply(newProtocol: DnsProtocol.Resolved): Resolved = {
+      Resolved(newProtocol.name, newProtocol.records.collect {
+        case r: ARecord    => r.ip
+        case r: AAAARecord => r.ip
+      })
+    }
   }
 
   /**
    * Lookup if a DNS resolved is cached. The exact behavior of caching will depend on
    * the akka.actor.io.dns.resolver that is configured.
    */
+  @deprecated("use cached(DnsProtocol.Resolve)", "2.6.0")
```

Comment:
Capitalize all of these with 
```suggestion
  @deprecated("Use cached(DnsProtocol.Resolve)", "2.6.0")
```